src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,6 @@
+.footer {
+  height: 400px;
```

Comment:
It's recommended to use an SCSS variable for the height value (e.g., `$footer-height: 400px;`) to improve maintainability and consistency. 🌟